server/src/modules/rules/rules.service.ts

```diff
@@ -299,14 +308,16 @@ export class RulesService {
           group.collectionId,
         );
 
-        // if datatype or manual collection settings changed then remove the collection media and specific exclusions. The Plex collection will be removed later by updateCollection()
+        // if datatype, manual collection settings or *arr server changed then remove the collection media and specific exclusions. The Plex collection will be removed later by updateCollection()
         if (
           group.dataType !== params.dataType ||
           params.collection.manualCollection !==
             dbCollection.manualCollection ||
           params.collection.manualCollectionName !==
             dbCollection.manualCollectionName ||
-          params.libraryId !== dbCollection.libraryId
+          params.libraryId !== dbCollection.libraryId ||
+          params.radarrSettingsId !== dbCollection.radarrSettingsId ||
```

Comment:
I've added a warning dialog when switching between arr server to alert the user that this will happen